plugins/node.yaml

```diff
@@ -0,0 +1,163 @@
+# Plugin Info
+version: 0.0.1
+title: Node.js
+description: Log Parser for Node.js
```

Comment:
Can you mention the log library being used, in this case "winston".

Reply:
https://github.com/winstonjs/winston

Reply:
Mentioned in the description. Did not add url as last I knew description is not displayed on site and in tooltips you can't have a url it will not display as a link.